dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/hadoop/LocalFilesystemScanCommandGenerator.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2022-2024 Google LLC
+ * Copyright 2013-2021 CompilerWorks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.edwmigration.dumper.application.dumper.connector.hadoop;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+import static java.util.stream.Collectors.joining;
+
+import com.google.common.base.Splitter;
+import com.google.common.escape.Escaper;
+import com.google.common.escape.Escapers;
+import com.google.common.io.Resources;
+import java.io.IOException;
+import java.net.URL;
+
+public class LocalFilesystemScanCommandGenerator {
+  private static final Escaper SIMPLE_BASH_ESCAPER =
+      Escapers.builder().addEscape('\'', "\\'").addEscape('\\', "\\\\").build();
+
+  public static String generate() {
+    try {
+      URL resourceUrl = Resources.getResource("hadoop-extraction/search-expressions.txt");
```

Comment:
The list looks small, would it be practical to move it here and remove the txt resource? This would avoid the need for custom parsing

Reply:
The size of the list doesn't matter. The point of the text resource file was to be able to edit the list comfortably without looking at the Java syntax. The custom parsing logic is only a few lines of code, so it doesn't affect anything significantly and it is likely that the parsing logic will be changed very rarely, while the expressions may be updated much more often.

I've changed the format to the java list.